libs/sdk-ui-dashboard/src/presentation/filterBar/attributeFilter/dashboardDropdownBody/configuration/hooks/useDependentDateFilterConfigurationState.ts

```diff
@@ -38,14 +39,47 @@ export function useDependentDateFilterConfigurationState(
                 to: neighborFilter.dateFilter.to,
                 granularity: neighborFilter.dateFilter.granularity,
                 type: neighborFilter.dateFilter.type,
+                isCommonDate: false,
             };
         });
-    }, [neighborFilters]);
+    }, [neighborFilters, filterElementsByDate]);
 
-    // TODO LX-157
-    const commonFilter = parseCommonDateFilter(commonDateFilter);
+    const commonDateFilters: IDashboardDependentDateFilter[] = useMemo(() => {
+        const commonDate = parseCommonDateFilter(commonDateFilter);
 
-    return [...neighborDateFiltersWithDimensions, commonFilter];
+        return (filterElementsByDate ?? [])
+            .filter((parent) => parent.isCommonDate === true)
```

Comment:
`parent.isCommonDate` should be enough? Possibly `!!parent.isCommonDate`?

Reply:
Updated it with `!!parent.isCommonDate`